SWIPs/swip-bzz-handshake.md

```diff
@@ -0,0 +1,78 @@
+bzz
+===
+
+The bzz protocol consists of a single message type; handshake.
+
+The bzz handshake is the first message to be exchanged between peers
+after the p2p transport protocol has been successfully negotiated. After
+the handshake each peer should remember the following data about the
+other:
+
+-   Swarm Overlay Address of the peer
+
+-   Whether the peer is a *light* or *full* node
+
+message
+-------
+
+The handshake message consists of four data fields:
+
+``` {numbers="none"}
+version     = UINT64
+Network ID  = UINT64
+Address data    = ADDRESSPAIR
+Light Node      = BOOL
+```
+
+The `AddressPair` consists of two strings, RLP-encoded as a single
```

Comment:
ADDRESSPAIR? Inconsistent

Reply:
With the data type definition in https://github.com/nolash/SWIPs/pull/4 we can delete these lines I think.